fcrepo-api-x-jena/src/main/java/org/fcrepo/apix/jena/impl/JenaOntologyService.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.fcrepo.apix.jena.impl;
+
+import static org.fcrepo.apix.jena.impl.Util.parse;
+
+import java.net.URI;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.fcrepo.apix.model.Ontology;
+import org.fcrepo.apix.model.WebResource;
+import org.fcrepo.apix.model.components.OntologyRegistry;
+import org.fcrepo.apix.model.components.OntologyService;
+
+import org.apache.jena.ontology.OntModel;
+import org.apache.jena.ontology.OntModelSpec;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.ModelGetter;
+import org.apache.jena.rdf.model.ModelReader;
+import org.apache.jena.rdf.model.RDFNode;
+import org.apache.jena.rdf.model.Resource;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * Uses Jena to parse and provide reasoning over ontologies.
+ */
+@Component(configurationPolicy = ConfigurationPolicy.REQUIRE)
+public class JenaOntologyService implements OntologyService {
+
+    private OntologyRegistry registry;
+
+    private OntModelSpec defaultSpec = OntModelSpec.OWL_MEM_MICRO_RULE_INF;
+
+    static final String RDF_TYPE = "http://www.w3.org/1999/02/22-rdf-syntax-ns#type";
+
+    static final String OWL_ONTOLOGY = "http://www.w3.org/2002/07/owl#Ontology";
+
+    static final String OWL_IMPORTS = "http://www.w3.org/2002/07/owl#imports";
+
+    public void setOntModelSpec(OntModelSpec spec) {
+        defaultSpec = new OntModelSpec(spec);
+        defaultSpec.setImportModelGetter(new NullGetter());
+    }
+
+    public JenaOntologyService() {
+        defaultSpec.setImportModelGetter(new NullGetter());
+    }
+
+    @Reference
+    public void setRegistryDelegate(OntologyRegistry registry) {
+        this.registry = registry;
+    }
+
+    @Override
+    public Ont getOntology(final URI uri) {
```

Comment:
@birkland probably because this line (unlike all the others) didn't change with that revision.